test/views/declaration/BorderTransportViewSpec.scala

```diff
@@ -82,10 +78,6 @@ class BorderTransportViewSpec extends UnitViewSpec with CommonMessages with Stub
       view.getElementById("title").text() mustBe messages("supplementary.transportInfo.title")
     }
 
-    "display header" in {
-      view.select("legend>h1").text() mustBe messages("supplementary.transportInfo.title")
```

Comment:
is this a regression ?

Reply:
No, this is no longer valid and it was being tested twice. Redundant